core/drivers/crypto/caam/caam_jr.c

```diff
@@ -642,7 +643,7 @@ void caam_jr_resume(uint32_t pm_hint)
 		caam_hal_jr_config(jr_privdata->baseaddr, jr_privdata->nb_jobs,
 				   jr_privdata->paddr_inrings,
 				   jr_privdata->paddr_outrings);
-#endif /* CFG_NXP_CAAM_RUNTIME_JR */
+	}
```

Comment:
I may be missing the point, but doesn't this change means that the `#ifndef CFG_NXP_CAAM_RUNTUME_JR` a few lines up should be replaced with a `if (!IS_ENABLED(CFG_NXP_CAAM_RUNTIME_JR))` ?

Reply:
This is a leftover commit which has AFAIR since been fixed by another commit, I will drop this on the next rebase.